doc/v2.rst

```diff
@@ -35,7 +35,88 @@ If you are executing this pip install command in a notebook, make sure to restar
 Changes
 *******
 
-To see what changes have been made, see the `CHANGELOG <https://github.com/aws/sagemaker-python-sdk/blob/zwei/CHANGELOG.md>`_.
+This section is for major changes that may require updates to your SageMaker Python SDK code.
+You can also see what changes have been made in the `CHANGELOG <https://github.com/aws/sagemaker-python-sdk/blob/zwei/CHANGELOG.md>`_.
```

Comment:
Why in the zwei branch?

Reply:
I see... this is going into that branch...

Reply:
yeah. everything will get merged into the main branch when v2.0 is finally released